### DISCUSSION TITLE
puanlama

### DISCUSSION THREAD

**Discussion by yagmurgul** [19/classroom#30] [Anonim] [answered]
![hata](https://user-images.githubusercontent.com/91977186/158448285-2d151ed5-16e3-4aa4-99b9-d176e941f89c.png)
![kod](https://user-images.githubusercontent.com/91977186/158448303-7d4fc2ce-7e73-46e5-8902-56d49f5705b5.png)

Kodun assignment.c de böyle görünmesine rağmen puanlanırken sağdaki kod gibi görüntülendiğini soyluyor.

**Comment by roktas** [comment, marked as answer]
Sekme (`\t`) ve boşluk arasındaki farkı araştır.

**Reply by yagmurgul** [reply]
Tesekkurler hocam

**Comment by roktas** [comment]
@yagmurgul Bir de lütfen cevabınızı (yanlış bile olsa) açık olarak yazmayın foruma.
